dev-libs/libqt5pas/libqt5pas-1.2.9.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 2020-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit qmake-utils
+
+LAZRELEASE="2.2.4"
+
+if [[ ${PV} = 9999* ]]; then
+	EGIT_REPO_URI="https://gitlab.com/freepascal.org/lazarus/lazarus.git"
+	inherit git-r3
+else
+	SRC_URI="https://sourceforge.net/projects/lazarus/files/Lazarus%20Zip%20_%20GZip/Lazarus%20${LAZRELEASE}/lazarus-${LAZRELEASE}-0.tar.gz/download -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+fi
```

Comment:
Detete everything and leave only
```
SRC_URI="https://sourceforge.net/projects/lazarus/files/Lazarus%20Zip%20_%20GZip/Lazarus%20${LAZRELEASE}/lazarus-${LAZRELEASE}-0.tar.gz/download -> ${P}.tar.gz"
KEYWORDS="~amd64 ~x86"
```
as you have not a live ebuild too

Reply:
I was hoping to use it in the next phase, but done.